app/src/org/commcare/dalvik/dialogs/PaneledChoiceDialog.java

```diff
@@ -89,4 +90,30 @@ public void setOnDismissListener(DialogInterface.OnDismissListener listener) {
     public void dismiss() {
         dialog.dismiss();
     }
+
+    public void addCollapsibleInfoPane(String messageContent) {
+        View extraInfoContainer = view.findViewById(R.id.extra_info_container);
+        extraInfoContainer.setVisibility(View.VISIBLE);
+
+        TextView extraInfoContent = (TextView)view.findViewById(R.id.extra_info_content);
+        extraInfoContent.setText(messageContent);
+
+        final ImageButton extraInfoButton = (ImageButton)view.findViewById(R.id.extra_info_button);
+        extraInfoButton.setVisibility(View.VISIBLE);
+        extraInfoButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                toggleExtraInfoVisibility();
+            }
+        });
+    }
+
+    private void toggleExtraInfoVisibility() {
+        TextView extraInfoContent = (TextView)view.findViewById(R.id.extra_info_content);
```

Comment:
exactly. Popping in text that changes the view size can be disorienting for people. It's good for us to smooth it over a bit.